docker/buildDocker.sh

```diff
@@ -99,51 +99,46 @@ checkArgs()
         	echo "WORKSPACE not found, setting it as environment variable 'HOME'"
         	WORKSPACE=$HOME
 	fi
-	if [[ "$useEclipseDockerFiles" == true && "$useEclipseDockerSlavesFiles" == true ]]; then
-		echo "Unable to use both kinds of dockerfiles at once, Select a single option."
-		exit 1
-        fi
+	if [ "$cleanWorkspace" == true ]; then
+		echo "Cleaning workspace"
+		rm -rf $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant
+	fi
 }
 
 useEclipseDockerFiles()
 {
-	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant && mkdir -p EclipseDockerfiles
-	cd EclipseDockerfiles
-	for jdk in $jdkVersion
-	do
-		# ${jdk%?} will remove the 'u' from 'jdk__u' when needed.
-		curl -o Dockerfile.$jdk https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/${jdk%?}/x86_64/ubuntu16/Dockerfile;
-		sharedEclipseDockerCommands $jdk
-	done
-}
-
-useEclipseDockerSlavesFiles()
-{
-	cd $WORKSPACE/DockerBuildFolder/
-	git clone https://github.com/eclipse/openj9 $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant 
-	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/buildenv/jenkins/docker-slaves/x86/centos6.9
-	if [ -f "known_hosts" ]; then 
-		rm known_hosts
-	fi
+	mkdir -p $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles
+	cd $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles
+	wget https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/mkdocker.sh
+	chmod +x mkdocker.sh
+	# Generate an Ubuntu1804 Dockerfile using mkdocker.sh
+	$WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles/mkdocker.sh --dist=ubuntu --version=18 --print >> $WORKSPACE/DockerBuildFolder/$jdkVersion-$buildVariant/EclipseDockerfiles/Dockerfile
```

Comment:
Do we need to parameterize that 18?

Reply:
For now, I'd rather not. While we _can_ make dockerfiles with different distributions and versions with OpenJ9's `mkDocker.sh` script, to keep it consistent with our own `dockerfile-generator.sh` script and for the purposes of `DockerfileCheck`, I think they should be all on Ubuntu1804. Extended `dockerfile-generator.sh` would certainly be a good future enhancement however